hydra-node/src/Hydra/API/WSServer.hs

```diff
@@ -92,22 +96,25 @@ wsApp party tracer history callback headStatusP headIdP snapshotUtxoP responseCh
     hydraHeadId <- atomically getLatestHeadId
     snapshotUtxo <- atomically getLatestSnapshotUtxo
     time <- getCurrentTime
-
-    sendTextData con $
-      Aeson.encode
-        TimedServerOutput
-          { time
-          , seq
-          , output =
-              Greetings
-                { me = party
-                , headStatus
-                , hydraHeadId
-                , snapshotUtxo
-                , hydraNodeVersion = showVersion Options.hydraNodeVersion
-                } ::
-                ServerOutput tx
-          }
+    case mapStateChangedToServerOutput
+      ( Greetings
+          { me = party
+          , headStatus
+          , hydraHeadId
+          , snapshotUtxo
+          , hydraNodeVersion = showVersion Options.hydraNodeVersion
+          } ::
+          StateChanged tx
+      ) of
+      Nothing -> pure ()
+      Just greetings ->
+        sendTextData con $
+          Aeson.encode
+            TimedServerOutput
+              { time
+              , seq
+              , output = greetings
+              }
```

Comment:
Why is this not just producing the `Greetings` message as before?

Consequently, there should not be a `Greetings :: StateChanged`